extensions/metadata.md

```diff
@@ -0,0 +1,739 @@
+---
+title: Metadata
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "Kiyoshi Aman"
+    period: "2012"
+    email: "kiyoshi.aman@gmail.com"
+  -
+    name: "Attila Molnar"
+    period: "2015-2016"
+    email: "attilamolnar@hush.com"
+  -
+    name: "James Wheare"
+    period: "2018"
+    email: "james@irccloud.com"
+  -
+    name: "Daniel Oaks"
+    period: "2021"
+    email: "daniel@danieloaks.net"
+  -
+    name: "Valerie Pond"
+    period: "2022"
+    email: "v.a.pond@outlook.com"
+  -
+    name: "Val Lorentz"
+    period: "2022"
+    email: "progval+ircv3@progval.net"
+---
+
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the
+unprefixed `metadata-2` or `metadata-notify-2` capability names.
+Instead, implementations SHOULD use the `draft/metadata-2` and
+`draft/metadata-notify-2` capability name to be interoperable with other
+software implementing a compatible work-in-progress version.
+
+The final version of the specification will use an unprefixed capability name.
+
+## Introduction
+
+It is useful to associate metadata with one's IRC presence, e.g. to
```

Comment:
Most of the spec is using soft newlines but this section is using hard newlines.